adding_species_manuscript.md

```diff
@@ -162,28 +164,113 @@ with stdpopsim.
 
 ## Making a popgen simulation
 
-To include a species into the stdpopsim catalog it is required that the species has at least: 
-
-- (1) a **genome assembly**, with contigs being assembled at the chromosome level or nearly so. By using chromosome level information to simulate genomes, stdpopsim indirectly simulates the effect of linked selection. [maybe discussion next text -> ] In the near future this won't be a problem for multiple species, since with (i) the advent of long-read sequencing technologies [ref] and because of (ii) initiatives such as B10K, XXXX, and G10K near XXX error-free reference genome assemblies [Rhie et al, 2021, REF, REF] spanning more than XXX vertebrate species are expected to be generated.  
-
-- (2) It is also required that the species has a plausible and citeable **mutation rate** estimate. Both phylogenetic mutation rate [ref], germline *de novo* mutations, or mutation rates estimates based on mutation accumulation studies [ref] are welcomed. [commment: using the mutation rate of a closely related species is not a big deal, either??? I know you guys use the drosophila one for the anopheles].
-
-- (3) Since both mutation and recombination shape the genetic diversity of genomes, stdpopsim also requires a **recombination rate** estimate for the species. Ideally, this should be a chromosome level recombination map, which allows more precise inference of the effect of selective interference. At minimum, a citeable single recombination rate estimate for the whole genome.
-
-- (4) A generation time estimate (in years).
-
-- (5) Population-size history also shapes the levels of diversity in a given population/genome. For this reason, to accurately simulate genomes, stdpopsim also expects that the species has a citeable **demographic model**, or at least a plausible and citeable **effective population size** (Ne).
-
-Users can also add other attributes of a species to the simulation that are not strictly required but can be useful, such as:
-
-- (6) **Genome annotation**, such as a GFF3/GFF format [ref] containing information about the coordinates of coding and noncoding regions; and the position of specific genes. 
-
-- (7) A citeable **distribution of fitness effects** (DFE, REF) for the species. This feature is particularly useful when simulating genomes under non-neutral models. 
-
-    - draw from https://github.com/popsim-consortium/workshops/blob/main/adding_species/contributing.ipynb without specifics for stdpopsim 
-    - emphasize throughout that these features should be citeable and chosen with deliberation **PETER**
-
-        - eg. "often reported parameter values are tied to a model and its other parameters, so mixing values from different sources has its own caveats" (https://github.com/popsim-consortium/adding-species-manuscript/issues/7)
+The purpose of including a species into the stdpopsim catalogue
```

Comment:
I think we want this part to be more general, not just referring to the stdpopsim catalog but making a good popgen simulation in general. Then we could use this paragraph as the introduction to section 6 (steps to add to the stdpopsim catalog), and edit the other stdpopsim references to reflect popgen simulations in general, and/or add references to section 6 below.

Reply:
So instead we could start this section with something like "Purposes of generating population genetic simulations include testing and benchmarking inference methods, creating data for Approximate Bayesian Computation and for training machine learning methods, and <... should complement the introduction ...>. Inference and methods testing improve the more aspects of realism can be incorporated into the simulations, so the effects of linkage, recombination rate variation, demographic changes, and other real-world factors affecting patterns of genetic variation in real species can be modeled. So when choosing to generate realistic population genetic simulations for a species of interest, certain elements should be known. We do not attempt here to create tutorials for any specific population genetic simulation programs (such as ms, msprime, ..., or SLiM), but to describe the necessary elements for generating realistic simulations for a species of interest with any of these programs. Concretely, all species would ideally have:"